packages/react-docsite-components/src/components/PlatformPicker/PlatformPicker.tsx

```diff
@@ -33,7 +33,7 @@ export class PlatformPicker<TPlatforms extends string = string> extends React.Pu
                     href={pages && this._getFirstPageUrl(pages)}
                     className={css(styles.button, platformKey === activePlatform && styles.activePlatform)}
                     iconProps={{
-                      iconName: icon,
+                      iconName: platform.name !== 'Web Components' ? icon : '',
```

Comment:
Can you add a comment about why web components doesn't have an icon here?

Reply:
```suggestion
                      iconName: platform.name !== 'Web Components' ? icon : '', // Hide icon to help label fit
```